test/ziggurat/config_test.clj

```diff
@@ -52,6 +52,15 @@
         (is (= (:rabbit-mq (:ziggurat config-values-from-env)) (rabbitmq-config)))
         (mount/stop)))))
 
+(deftest statsd-config-test
```

Comment:
I think we should have more tests here for these cases:
1) only datadog config In config.edn
2) only statsd config In config.edn
3) both datadog and statsd config In config.edn

Reply:
done